opensrp-app/src/test/java/org/smartregister/service/HTTPAgentTest.java

```diff
@@ -103,6 +104,7 @@ public void testUrlCanBeAccessWithGivenCredentialsGivenWrongUrl(){
     }
 
     @Test
+    @Ignore
```

Comment:
Why is this being ignored

Reply:
An issue has been created for fixing that test here https://github.com/OpenSRP/opensrp-client-core/issues/511